Tools/_lust/replace_nt.py

```
@@ -7,6 +7,8 @@
     r"\bNT\b": "Qillu",
     r"\bНТ\b": "Qillu",
     r"\bНанотрейзен\b": "Qillu",
+    r"\bNanoTrasen\b": "Qillu",
+    r"\bНаноТрейзен\b": "Qillu",
```

Ruff (0.12.2)

String contains ambiguous `Н` (CYRILLIC CAPITAL LETTER EN). Did you mean `H` (LATIN CAPITAL LETTER H)?

(RUF001)